ansible/provision-sft.yml

```diff
@@ -7,5 +7,8 @@
     - role: sft-monitoring-certs
 
 - hosts: sft_servers
-  roles:
-    - role: sft-server
+  tasks:
+    - import_role:
+        name: 'sft-server'
+    - import_role:
+        name: 'srv-announcer'
```

Comment:
Is there a difference between listing roles and importing them?

Reply:
Technically speaking, no - I remembered it differently. I thought, `roles:` would load them dynamically, but [I was wrong](https://docs.ansible.com/ansible/2.7/user_guide/playbooks_reuse.html#differences-between-static-and-dynamic). It's loading them statically. Writing it this way, it makes it more obvious, though. Anyway, I'll change it back.